### PR TITLE
Disable map's mouse handling while scaling sections are handling the …

### PR DESCRIPTION
**Pull request by gokaysatir**
…mouse events.

Change-Id: Ib7a8e9a43ce8493d23d5b55e355d66c3a24cbba1

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by eszkadev** [comment]
This PR fixes resize handles for me, but rotation one is still not working

**Comment by eszkadev** [comment]
ci fail:
```
cy:command ✔  cGet	#map .html-object-section
      cy:command ✘  assert	expected **<div.html-object-section>** to have a length of **9** but got **1**
      cy:command ✔  fail:	
                    Test failed: integration_tests/desktop/impress/editable_area_spec.js / Editable area - Basic typing and caret moving / Deleting text
 ```

**Comment by gokaysatir** [comment]
> ci fail:
> 
> ```
> cy:command ✔  cGet	#map .html-object-section
>       cy:command ✘  assert	expected **<div.html-object-section>** to have a length of **9** but got **1**
>       cy:command ✔  fail:	
>                     Test failed: integration_tests/desktop/impress/editable_area_spec.js / Editable area - Basic typing and caret moving / Deleting text
> ```

Hmm, let me check this. I improved the usage for also rotation and custom sections.

**Comment by gokaysatir** [comment]
I updated the failing test. It should pass now.

**Comment by mmeeks** [comment]
@timar I guess a prioirty to get this merged and included in the green release =)

**Comment by timar** [comment]
> @timar I guess a prioirty to get this merged and included in the green release =)
Review required

File: cypress_test/integration_tests/desktop/impress/editable_area_spec.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ function selectTextShape(i) {
             cy.cGet('body').click(XPos, YPos);
         });
 
-    cy.cGet('#map .html-object-section').should('have.length', 9);
+    cy.cGet('[id^="test-div-shape-handle-"]').should('have.length', 6); // 3 of the handles are not visible because of the side bar.
```
Comment (eszkadev) [inline cypress_test/integration_tests/desktop/impress/editable_area_spec.js:25]: hmm, wouldn't be better to hide sidebar to be sure we are having all the needed handles?

**Review by eszkadev — APPROVED** [review]

**Pull request merged**